driver/src/contracts/stablex_contract.rs

```diff
@@ -38,9 +38,9 @@ pub trait StableXContract {
     fn get_auction_data(&self, _index: U256) -> Result<(AccountState, Vec<Order>)>;
     fn submit_solution(
         &self,
-        _batch_index: U256,
-        _orders: Vec<Order>,
-        _solution: Solution,
+        batch_index: U256,
```

Comment:
At this moment we do not. However, I have already noticed that this will necessarily need to be changes to `u32` anyways for the purpose of comparing with `order.validTo` and `order.ValidFrom` when filtering out the `EncodedAuctionElements`. Thanks for pointing this out.